src/ui/actions/timeline.ts

```diff
@@ -256,11 +237,15 @@ export function seek({
 
     assert(executionPoint != null, `Could not find execution point for time ${time}`);
 
-    if (pauseId) {
-      ThreadFront.timeWarpToPause({ point: executionPoint, time, pauseId }, openSource);
-    } else {
-      ThreadFront.timeWarp(executionPoint, time, openSource);
-    }
+    dispatch(paused({ executionPoint, time, openSource }));
+
+    const focusWindow = replayClient.getCurrentFocusWindow();
```

Comment:
:speech_balloon: possibly answering my own question, looks like it ended up down here?

Reply:
Exactly.